SConstruct

```diff
@@ -2461,14 +2461,14 @@ if doConfigure :
 
 houdiniEnvSets = {
 	"IECORE_NAME" : "IECoreHoudini",
-	"CXXFLAGS" : dependencyIncludes + [ "$HOUDINI_CXX_FLAGS", "-DMAKING_DSO" ]
```

Comment:
Just checking I understand : it's this change from set to append that means the `-fvisibility=hidden` compiler flag gets set when building IECoreHoudini, correct?

Reply:
Yes, that's correct. As well as the other flags we were setting above like `-Werror`, the BUILD_TYPE flags, extra ASAN flags, etc.